Mathlib/Algebra/Category/ModuleCat/Presheaf/Pushforward.lean

```diff
@@ -28,15 +28,16 @@ namespace PresheafOfModules
 
 variable (F : C ⥤ D)
 
+set_option maxHeartbeats 400000 in -- Seems to be a timeout in type checking the definition
```

Comment:
On my machine, type checking the definition costs 2,5 seconds, about a third of total elaboration time. I haven't been able to trace where all the extra time goes...